docs/adoc/releasenotes/ReleaseNotes_9_0.adoc

```diff
@@ -178,3 +178,42 @@ The dogs bark
 ----
 
 Projects that wish to keep the existing behavior can do so by providing their own `CollatorProvider` (see migration guide).
+
+== New properties for MenuBar design
```

Comment:
Please use "title case" for titles (https://english.stackexchange.com/questions/14/)